packages/providers/src/util.test.ts

```diff
@@ -71,4 +71,45 @@ describe('Providers utils', () => {
       'https://explorer.fuel.sh/transaction/0x123?providerUrl=https://rpc.fuel.sh'
     );
   });
+
+  test('buildBlockExplorerUrl - helper params', () => {
+    // passing in multiple helper params should throw
+    expect(() =>
+      buildBlockExplorerUrl({
+        blockExplorerUrl: 'explorer.fuel.sh',
+        providerUrl: 'rpc.fuel.sh',
+        address: '0x123',
+        txId: '0x123',
+        blockNumber: 123,
+      })
+    ).toThrow();
```

Comment:
Please, try always to match something about the thrown Error message to ensure that the error being thrown is the expected one and not another (_sometimes, even unrelated_) error. This can be useful against false positives.

You can use strings or regexes for this.

```suggestion
    ).toThrow(/My pretty regex here/);
```